components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -878,5 +879,79 @@ namespace scream {
             });
         }
 
+        void compute_aerocom_cloudtop(
```

Comment:
See below comment, but this could probably be impl’d in Kokkos and avoid some extra copies back and forth between YAKL arrays and Kokkos views. I.e., could probably use Kokkos views directly from the Field Manager rather than relying on the YAKL copies.